pkg/asset/installconfig/basedomain.go

```diff
@@ -36,8 +36,13 @@ func (a *baseDomain) Generate(parents asset.Parents) error {
 	switch platform.CurrentName() {
 	case aws.Name:
 		var err error
-		a.BaseDomain, err = awsconfig.GetBaseDomain()
+		zone, err := awsconfig.GetBaseDomain()
+		if err != nil {
+			return err
+		}
 		cause := errors.Cause(err)
+		a.BaseDomain = zone.DNSName
+		platform.AWS.PublicZoneID = zone.ID
 		if !(awsconfig.IsForbidden(cause) || request.IsErrorThrottle(cause)) {
 			return err
 		}
```

Comment:
previous to this change, if there was `Forbidden / Throttle error` when fetching the r53 dns zones from API we would automatically fall back to just prompting the use for base domain string.

i think this changes that behavior, would like to keep that previous behavior.